packages/core/src/runner/helper.ts

```diff
@@ -11,6 +11,8 @@ import { Observable, from } from 'rxjs';
 import { flatMap } from 'rxjs/operators';
 import { DATA, MODEL, EVALUATE, DEPLOYMENT, MODELTOSAVE } from '../constants/other';
 
+const debug = require('debug')('core/runner');
```

Comment:
See https://stackoverflow.com/questions/47724165/how-to-use-the-debug-module-with-typescript, change this to:

```ts
import Debug from 'debug';
const debug = Debug('pipcook/core/runner');
```